CHANGELOG.md

```diff
@@ -81,6 +81,10 @@ BUG FIXES:
 
 ## 2.5.0 (March 30, 2020)
 
+BREAKING CHANGES:
+
+* `go-github` `v29.03` is incompatible with all currently known versions of GitHub Enterprise Server ([[#404](https://github.com/terraform-providers/terraform-provider-github/issues/404)])
```

Comment:
looks like this could be updated based on the issue comments https://github.com/terraform-providers/terraform-provider-github/issues/404#issuecomment-694367840

Reply:
Hopefully, it will be confirmed shortly.

Reply:
Just getting caught up on this one.  Admittedly, compatible GHES versions are something that our testing is blind to and am hoping this changes going forward.  From the linked issue, it seems like:

> 2.9.2 works fine for us with GHES 2.21.5

> GitHub Enterprise v2.20.3 is compatible only up to v2.4.1 of this provider.

Is there a more accurate statement we can make here? Maybe this?

```suggestion
* `go-github` `v29.03` is incompatible with versions of GitHub Enterprise Server prior to `v2.21.5`. ([[#404](https://github.com/terraform-providers/terraform-provider-github/issues/404)])
```